init.lua

```diff
@@ -801,7 +801,7 @@ require('lazy').setup({
       --
       -- Examples:
       --  - va)  - [V]isually select [A]round [)]paren
-      --  - yinq - [Y]ank [I]nside [N]ext [']quote
+      --  - yin' - [Y]ank [I]nside [N]ext [']quote
```

Comment:
@VlaDexa @syhner Perhaps then, the comment should instead be changed to:
```lua
      --  - yinq - [Y]ank [I]nside [N]ext [Q]uote
```
to match the use of `q` instead of `'` - thoughts?

Reply:
Opened https://github.com/nvim-lua/kickstart.nvim/pull/985